ShareMyOffer/src/main/java/com/neeraja/ShareMyOffer/entities/Address.java

```diff
@@ -65,16 +68,13 @@
 	@Column(name = "pincode")
 	private int pinCode;
 	
-	@Null
 	@Column(name = "latitude")
 	private double latitude;
 	
-	@Null
 	@Column(name = "longitude")
 	private double longitude;
 	
-	public Address(int id, String houseNumber,String landMark,String street,String city,String state,String country,int pincode,double latitude,double longitude,User user) {
-		this.id = id;
+	public Address(String houseNumber,String landMark,String street,String city,String state,String country,int pincode,double latitude,double longitude,User user) {
```

Comment:
We should not have constructors in entites. Either use constructor builder or setters methods.

Reply:
Updated it!